.yamato/training-int-tests.yml

```diff
@@ -0,0 +1,27 @@
+test_editors:
+  - version: 2018.4
+  - version: 2019.3
+---
+{% for editor in test_editors %}
+test_mac_editmode_{{ editor.version }}:
+  name: Test Mac Fast Training {{ editor.version }}
+  agent:
+    type: Unity::VM::osx
+    image: ml-agents/ml-agents-bokken-mac:v0.1.3-build-tensorflow-489268
```

Comment:
This is using the image built by the PR (https://gitlab.cds.internal.unity3d.com/ml-agents/bokken-images/ml-agents-bokken-mac/merge_requests/1). I can merge that and use an "official" image if that's better.

Reply:
should probably merge it.

Reply:
Merged. Will figure out the tag to use tomorrow